Formula/inlets.rb

```diff
@@ -0,0 +1,33 @@
+class Inlets < Formula
+  desc "Expose your local endpoints to the Internet"
+  homepage "https://github.com/alexellis/inlets"
+  url "https://github.com/alexellis/inlets.git",
+      :tag      => "0.5.7",
+      :revision => "4c5a519b669cfb92d3ced715fc8e82489aded46a"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_OS"] = "darwin"
+    ENV["XC_ARCH"] = "amd64"
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/alexellis/inlets").install buildpath.children
+    cd "src/github.com/alexellis/inlets" do
+      commit = Utils.popen_read("git", "rev-parse", "HEAD").chomp
+      system "go", "build", "-ldflags",
+             "-s -w -X main.GitCommit=#{commit} -X main.Version=#{version}", "-a",
```

Comment:
Could you split these commands and maybe use the long form if it exists?
```suggestion
             "-s", 
             "-w",
              "-X main.GitCommit=#{commit}",
              "-X main.Version=#{version}",
              "-a",
```

Reply:
I will split the lines but this is the idiomatic Golang version of the flags, these are used as a de-facto standard and I don't want to maintain two sets of flags upstream and here.